library/src/blas3/rocblas_trsm.cpp

```diff
@@ -2,6 +2,7 @@
  * Copyright 2019 Advanced Micro Devices, Inc.
```

Comment:
Hmm, I think these changes are just from my trsm PR and don't need to be added in this one, hopefully?

Reply:
yes but tritri_trsm.hpp is modified to get this working, I will wait for your PR to get in first and then handle the merge. There shouldn't be many conflicts.